[ADD] website_sale_category_description

yajo opened the pull request:
With this module, you'll be able to add a description individually for each of your website product categories.

@Tecnativa TT23930

pedrobaeza commented on website_sale_category_description/models/product_public_category.py:
```diff
@@ -0,0 +1,17 @@
+class ProductPublicCategory(models.Model):
+    _inherit = "product.public.category"
+
+    website_description_1 = fields.Html(
```
Why not `website_description_top` (and `_bottom`)?

yajo replied:
I thought it but felt like not anchoring the field to a place would leave more room for theme designs, if some wanted to place these different.

But I'm not reluctant to name them like that if you think it's better.

pedrobaeza replied:
Well, their intention is clear, and designs can vary the exact place on top or on bottom, but to misuse them for putting for example both on top, or one in the middle, has no sense on the context of this module. If they do that, it can be still with the same fields. On the contrary, having `_1` and `_2`, you can have the doubt of which one represents the top part.

pedrobaeza commented on website_sale_category_description/readme/CONTRIBUTORS.rst:
```diff
@@ -0,0 +1 @@
+* Jairo Llopis <jairo.llopis@tecnativa.com> (`Tecnativa <https://www.tecnativa.com/>`__)
```
We use to put company and indented, the person (without the email for avoiding "spam").

chienandalu reviewed (approved):
Cool :+1:

rafaelbn commented on website_sale_category_description/README.rst:
```diff
@@ -0,0 +1,105 @@
+#. Enable *eCommerce Categories* if it was disabled.
+#. Navigate to any category.
+#. Click on *Edit*.
+#. You will see 2 block drop areas at the top, and 2 on the bottom:
```
I don't see 4 block drom areas, I see 2
![2020-05-15_17-41-31](https://user-images.githubusercontent.com/8736623/82069286-64b3eb00-96d3-11ea-8006-e9b411b87fc9.jpg)

rafaelbn commented on website_sale_category_description/README.rst:
```diff
@@ -0,0 +1,105 @@
+#. Click on *Edit*.
+#. You will see 2 block drop areas at the top, and 2 on the bottom:
+
+   * 1st top block is a description that will appear in all your eCommerce pages.
```
I have tested and this block doesn't exists

rafaelbn commented on website_sale_category_description/README.rst:
```diff
@@ -0,0 +1,105 @@
+#. You will see 2 block drop areas at the top, and 2 on the bottom:
+
+   * 1st top block is a description that will appear in all your eCommerce pages.
+   * 2nd top block is the same, but specific for this category.
```
:+1:

rafaelbn commented on website_sale_category_description/README.rst:
```diff
@@ -0,0 +1,105 @@
+
+   * 1st top block is a description that will appear in all your eCommerce pages.
+   * 2nd top block is the same, but specific for this category.
+   * 1st bottom block is also a specific description for this category.
```
:+1:

rafaelbn commented on website_sale_category_description/README.rst:
```diff
@@ -0,0 +1,105 @@
+   * 1st top block is a description that will appear in all your eCommerce pages.
+   * 2nd top block is the same, but specific for this category.
+   * 1st bottom block is also a specific description for this category.
+   * 2st bottom block is another description that will appear in all your eCommerce pages.
```
I have tested and this block doesn't exists

rafaelbn reviewed (dismissed):
Functional reviewed, check my comments please.

Commit:
[ADD] website_sale_category_description

With this module, you'll be able to add a description individually for each of your website product categories.

@Tecnativa TT23930

yajo commented:
@pedrobaeza @rafaelbn all fixed.

yajo commented:
Merge?

rafaelbn commented:
@Yajo I'm not able to save any kind of block in the botton, see this screenshot:

![2020-05-25_18-59-58](https://user-images.githubusercontent.com/8736623/82831951-1c0de600-9eba-11ea-8f1b-50f16581a359.png)

yajo commented:
@rafaelbn That issue is a bug in odoo framework fixed in https://github.com/odoo/odoo/pull/51885 that was affecting this module in https://github.com/OCA/e-commerce/commit/3fd89a2502480526be1f24df3c1bbe6ad033c07d.

I force-pushed it to reduce the feature scope and make it match what odoo v13 upstream does. Now it's not affected by that bug, as a side effect.

Here's how it looks like now:

![2020-05-27_08-21](https://user-images.githubusercontent.com/973709/82990402-0ac8f480-9ff4-11ea-8a9d-0cb2a59ced9c.png)

pedrobaeza commented:
/ocabot merge nobump
